pkg/descriptions.json

```diff
@@ -71,5 +71,10 @@
             "System.ServiceModel.UpnEndpointIdentity",
             "System.ServiceModel.SpnEndpointIdentity"
         ]
+    },
+    {
+        "Name": "svcutilcore",
+        "Description": "Generates data types that use the XmlSerializer for serialization and deserialization",
```

Comment:
Maybe we can "borrow" from `svcutil /t:xmlserializer` and have something like:

Generate serialization types for XmlSerializer types used by any Service Contracts in an assembly.